spec/feature/books/show_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe 'books#show', type: :feature do
+  let(:book) { create(:book) }
+
```

Comment:
pls add specs with increasing count of books or you can add it with specs on cart